### ISSUE TITLE
New Rule: `svelte/no-dupe-on-directives` and `svelte/no-dupe-use-directives`

### ISSUE THREAD

**Issue by ota-meshi** [ota-meshi/eslint-plugin-svelte#304]
### Motivation

We can define any number of `on:` and `use:` directives with the same name, but duplicate directives with the exact same name and expression are probably a mistake.
I would like to find it in the new rule.

See also https://github.com/sveltejs/svelte/pull/8048.

### Description

The new rule reports `on:` directives with exactly the same name and expression, and `use:` directives with exactly the same name and expression.

### Examples

```svelte
<!-- ✓ GOOD -->
<button on:click on:click={myHandler}  />
<button on:click={foo} on:click={bar}  />
<div use:clickOutside use:clickOutside={param}  />
<div use:clickOutside={foo} use:clickOutside={bar} />

<!-- ✗ BAD -->
<button on:click on:click />
<button on:click={myHandler} on:click={myHandler} />
<div use:clickOutside use:clickOutside  />
<div use:clickOutside={param} use:clickOutside={param}  />
```

### Additional comments

As for `on:`, maybe some users want to report even if the expression is different. I think we can add an option to the rule in that case.

```svelte
<!-- maybe BAD? -->
<button on:click={foo} on:click={bar}  />

<!-- GOOD? -->
<script>
function handler(evt) {
  foo(evt);
  bar(evt);
}
</script>
<button on:click={handler} />
```
